.github/workflows/ci.yml

```diff
@@ -18,30 +18,36 @@ jobs:
             - name: Setup Python
               uses: actions/setup-python@v2
               with:
-                  python-version: 3.8
+                  python-version: '3.9'
 
             - uses: pre-commit/action@v2.0.0
 
     test-app:
 
         needs: [pre-commit]
 
-        runs-on: ubuntu-latest
-        timeout-minutes: 10
-
         strategy:
             matrix:
-                tag: [stable]
-                browser: [chrome, firefox]
+                tag: [edge]
```

Comment:
The tag is not used in this file anymore.

Reply:
I keep it since going to use it as an ENV_VAR for using in the docker-compose. I'll change the docker-compose file.